assets/js/script.js

```diff
@@ -40,17 +44,25 @@ createNote = () =>{
     }else{
         warning.text("No note added yet 😉 ")
     }
-
     noteList.append("<li id='note_"+noteObject.id+"'>"+noteObject.body+"</li>");
     $("#note_"+noteObject.id).hide();
     $("#note_"+noteObject.id).fadeIn();
-
+    //empty textarea when note is added
+     var not = $.trim($('#note').val(''))
     console.log(notes);
 
+
     // store
     localStorage.setItem("notesStore", JSON.stringify(notes));
+   
+} else if (noteText=== ''){
+    console.log('warning, warning, empty note')
+    warning.text('Cannot add empty note 😉');
```

Comment:
Can you please make this a separate field? like, make this message to appear in red beneath the text area not in some field as the warning. Thanks

Reply:
Okay